### DISCUSSION TITLE
Como gerar QR Dinâmico sem imput de dados do cliente (ex: maquininhas cielo)

### DISCUSSION THREAD

**Discussion by felipesalomao** [bacen/pix-api#500] [Q&A]
Oi, estamos desenvolvendo uma solução pix, porém nos deparamos com um obstáculo. É obrigatório informar o cpf do pagador para gerar o pix dinâmico. Queremos implementar o pix em uma maquininha POS própria. Porém as maquininhas tipo cielo, rede, stone, etc não requer input de dados do pagador.. Simplesmente gera o qr code do pix..  Como posso fazer o mesmo? Gerar um qr dinâmico sem cpf do pagador?

**Comment by rubenskuhl** [comment]
> Oi, estamos desenvolvendo uma solução pix, porém nos deparamos com um obstáculo. É obrigatório informar o cpf do pagador para gerar o pix dinâmico. Queremos implementar o pix em uma maquininha POS própria. Porém as maquininhas tipo cielo, rede, stone, etc não requer input de dados do pagador.. Simplesmente gera o qr code do pix.. Como posso fazer o mesmo? Gerar um qr dinâmico sem cpf do pagador?

O QR-Code dinâmico não requer informação do pagador para ser gerado, ela é opcional. Apenas se uma parte for presente (ex:nome) que a outra parte também tem que estar presente (ex: CPF). Se gerar sem nome e sem CPF, funciona. A grande maioria dos QRs que vejo por aí são assim.
